help/sources/tutorials/ui/create/advertising/linkedin-ads.md

```diff
@@ -0,0 +1,150 @@
+---
+keywords: Experience Platform;home;popular topics;sources;connectors;source connectors;sources sdk;sdk;SDK
+title: Create a LinkedIn Ads source connection in the UI
+description: Learn how to create a LinkedIn Ads source connection using the Adobe Experience Platform UI.
+hide: true
+hidefromtoc: true
+---
+# Beta Create a [!DNL LinkedIn Ads] source connection in the UI
+
+>[!NOTE]
+>
+>The [!DNL LinkedIn Ads] source is in beta. See the [sources overview](../../../../home.md#terms-and-conditions) for more information on using beta-labelled sources.
+
+This tutorial provides steps for creating a [!DNL LinkedIn Ads] source connector using the Platform user interface.
+
+## Overview
+
+[[!DNL LinkedIn Ads]](https://business.linkedin.com/marketing-solutions/ads) is a marketing solution by LinkedIn to reach over 830 million professionals on the world's largest professional network. The LinkedIn Reporting & ROI APIs provide key insights on performance such as clicks, impressions, and ad spend, and demographics information such as metrics by demographic values at the account, campaign, and creative levels.
```

Comment:
Same as API:
Let's rephrase the first sentence to: "...is a marketing solution by LinkedIn that can reach over 830..."
And a slight restructuring to the second sentence: "...key insights on performance such as clicks, impressions, and ad spend, as well as demographic information like metrics by demographic values at the account, campaign, and creative levels."